### ISSUE TITLE
Analyzer: Improving spectrograph readability?

### ISSUE THREAD

**Issue by unfa** [calf-studio-gear/calf#27]
I find spectrographs very hard to read with the light-green backgrounds. I'd love to be able to make the backgrounds very dark and the displays yellow or green or RGB with full color spectrum used for the data,, I think the stylisation/functionality is here tilted too much left (towards stylisation).

![calf](https://cloud.githubusercontent.com/assets/491704/9049076/9041d7b0-3a3e-11e5-80ff-6fd47dd4dc48.png)

Maybe there's a way to change the stylisation, not hurting it, but improve the functionality?

Let's take a look at some other spectrograms present in Linux Audio:

Here's Audacity: it has a light gray background which isn't optimal, but uses colour information to improve the readability of the image:

![audacity](https://cloud.githubusercontent.com/assets/491704/9048934/4f8058b0-3a3d-11e5-9c81-8159698ef082.png)

This is FreqTweak, it uses both black background and color coding:

![freqtweak](https://cloud.githubusercontent.com/assets/491704/9048978/ad933350-3a3d-11e5-9ccd-1eb77a74e738.png)

And finally here's Sonic Visualiser:

![sonicvisualiser1](https://cloud.githubusercontent.com/assets/491704/9049032/3998b44c-3a3e-11e5-8f52-9e895db87e7a.png)

**Comment by jrigg** [comment]
I agree with unfa here. I appreciate that you're going for a certain aesthetic, but in a technical tool like this functionality should take precedence IMO.
